gramps/plugins/tool/verify.py

```diff
@@ -547,7 +546,9 @@ def process_family(
         yngmar,
         oldmar,
         oldmom,
+        olddad,
         yngmom,
+        yngdad,
```

Comment:
Two examples of undefined variables that happened because `globals()` modifications set them, instead of passing them as parameters.